### DISCUSSION TITLE
Why current balance is considering past transactions prior to initial balance?

### DISCUSSION THREAD

**Discussion by khorchanov** [firefly-iii/firefly-iii#9333] [Firefly III]
![IMG_20241008_092406.jpg](https://github.com/user-attachments/assets/6b558b40-41a9-4393-9fb9-e18c5523b720)

The current balance on this is shown as 70€ while I think it should be 80€

**Comment by JC5** [comment]
Initial balance is just another transaction. It doesn't cancel out anything you put in the system before the initial balance.
